activesupport/test/core_ext/time_with_zone_test.rb

```diff
@@ -482,22 +482,23 @@ def test_freeze_preloads_instance_variables
   end
 
   def test_method_missing_with_non_time_return_value
-    @twz.time.expects(:foo).returns('bar')
+    @twz.time.class.send(:define_method, :foo, -> { 'bar' })
```

Comment:
The class and `define_method` combo reads weird to me. What do you think of: `def @twz.time.foo; 'bar'; end`?